manuscript/teaching-reproducibility.Rmd

```diff
@@ -392,16 +392,21 @@ We then ask them to try to run it,
 and if they cannot run it, we ask them to fix the code 
 or install missing software so they can.
 Then we provide them the same analysis that has been made reproducible, 
-through the use of shareable compute environments (i.e., `renv`, `conda`, or Docker),
+through the use of shareable compute environments (i.e., `renv`, `conda`, or `Docker`),
 and instructions on how to use the shared compute environment.
 The students then experience running the same analysis on their machines 
 without any change of code or software installation.
 Under these controlled circumstances
 failure and frustration can have a positive impact on students overall learning
 as they experience the many benefits of reproducibility first hand.
-In a relatively short amount of time,
-this helps provide motivation
+This exercise usually only takes around 20 min
```

Comment:
Just double checking that 20 min is indeed the correct number here, it seems that it would take longer to go through failing and succeeding for a workflow.

Reply:
Haha, maybe! I am not as good as Mike in timing these things... We could say 20-30 min?